cmd/podman/buildfarm/create.go

```diff
@@ -0,0 +1,83 @@
+package buildfarm
+
+import (
+	"fmt"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/common/pkg/config"
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/containers/podman/v4/cmd/podman/utils"
+	"github.com/containers/podman/v4/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+var (
+	buildfarmCreateDescription = `Create a new farm with connections added via podman system connection add.
+
+	The "podman system connection add --farm" command can be used to add a new connection to a new or existing farm.`
+
+	createCommand = &cobra.Command{
+		Use:               "create [options] NAME [CONNECTIONS...]",
+		Args:              cobra.MinimumNArgs(1),
+		Short:             "Create a new farm",
+		Long:              buildfarmCreateDescription,
+		RunE:              create,
+		ValidArgsFunction: completion.AutocompleteNone,
+		Example: `podman buildfarm create myfarm connection1
+  podman buildfarm create myfarm`,
+	}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Command: createCommand,
+		Parent:  buildfarmCmd,
+	})
+	flags := createCommand.Flags()
+	flags.SetNormalizeFunc(utils.AliasFlags)
+}
+
+func create(cmd *cobra.Command, args []string) error {
+	farmName := args[0]
+	connections := args[1:]
+
+	cfg, err := config.ReadCustomConfig()
+	if err != nil {
+		return err
+	}
+
+	if _, ok := cfg.Farms.List[farmName]; ok {
+		// if farm exists return an error
```

Comment:
Should we add a --replace option?

Reply:
we could, but we already have `buildfarm update` that let's you add or remove connections from a farm and `buildfarm rm` can be used to remove a farm completely to create a new one.
I don't mind adding it, just think there are already other ways of doing a replace.

Reply:
I suggest to go ahead and implement the core functionality.  We can always improve UX at a later point.  At the moment, the core functionality of actually using buildfarm is missing and I think we need to make progress there first.